### PR TITLE
[ES] Heroes

### PR DESCRIPTION
**Pull request by luischos**
heroes1
heroes2
heroes3

### CODE REVIEW

File: translations/heroes.tex/es.po [hunk 1]
```diff
@@ -80,6 +84,14 @@ msgid ""
 "Gaining 1 Experience\\index{Experience} is represented by the symbol \\includesvg[height=10px]{\\svgs/experience.svg}.\n"
 "\n"
 msgstr ""
+"\\subsection*{Héroe Principal}\n"
+"El Héroe Principal\\index{Main Hero} está representado por su Figura, su Carta de Héroe y su Mazo.\n"
+"El Héroe Principal tiene 3 Puntos de Movimiento.\n"
+"Sólo el Héroe Principal puede usar el Mazo.\\par\n"
+"Cada Héroe Principal comienza el juego en el Nivel 1 y puede avanzar hasta el Nivel 7 ganando Experiencia.\n"
+"La experiencia se adquiere \\hyperlink{Combatexperience}{ganando Combates}, visitando ciertas \\hyperlink{All Map Locations}{Localizaciones} o con el \\hyperlink{Resources}{Dado de Tesoro} \\includesvg[height=10px]{\\svgs/treasure.svg}.\n"
+"Ganar 1 de experiencia\\index{Experience} se representa con el símbolo \\includesvg[height=10px]{\\svgs/experience.svg}.\n"
```
Comment (qwrtln) [inline translations/heroes.tex/es.po:93]: Please translate entries under `index{...}`.

File: translations/heroes.tex/es.po [hunk 2]
```diff
@@ -101,6 +113,21 @@ msgid ""
 "They can be Recruited again with another use of the Population Token.\\par\n"
 "\n"
 msgstr ""
+"\\subsection*{\\hypertarget{Secondary}{Héroe Secundario}}\n"
+"Si controlas una Ciudad o un Asentamiento, puedes contratar un Héroe Secundario\\index{Secondary Hero} volteando la \\textbf{Ficha de Población} y pagando 10 de Oro \\includesvg[height=10px]{\\svgs/gold.svg}.\\par\n"
```
Comment (qwrtln) [inline translations/heroes.tex/es.po:117]: Please translate entries under `index{...}`.

File: translations/heroes.tex/es.po [hunk 3]
```diff
@@ -151,40 +178,46 @@ msgid ""
...
-msgstr ""
+msgstr "\\textbf{Conocimiento} – Número de cartas de Conocimiento en tu Mazo Inicial"
 
 #. type: enumerate
 #: sections/heroes.tex:70
 msgid ""
 "\\textbf{Starting Ability} – Reminder for the unique Ability Card"
 "\\index{Ability Card} the Hero starts with."
 msgstr ""
+"\\textbf{Habilidad inicial} – Recordatorio de la Carta\\index{Ability Card} de Habilidad única "
```
Comment (qwrtln) [inline translations/heroes.tex/es.po:219]: Please translate entries under `index{...}`.

File: translations/heroes.tex/es.po [hunk 4]
```diff
@@ -307,6 +364,40 @@ msgid ""
 "\\end{itemize}\n"
 "\n"
 msgstr ""
+"\\subsection*{\\hypertarget{Level}{Efectos de subir de Nivel}}\n"
+"Los Héroes Principales siempre comienzan cada Escenario en el Nivel 1\\index{Level} y pueden subir de Nivel ganando Experiencia \\includesvg[height=10px]{\\svgs/experience.svg}.\n"
```
Comment (qwrtln) [inline translations/heroes.tex/es.po:368]:
```suggestion
"Los Héroes Principales siempre comienzan cada Escenario en el Nivel 1\\index{Nivel} y pueden subir de Nivel ganando Experiencia \\includesvg[height=10px]{\\svgs/experience.svg}.\n"
```

File: translations/heroes.tex/es.po [hunk 5]
```diff
@@ -307,6 +364,40 @@ msgid ""
...
+"\\item Su límite de mano \\includesvg[height=10px]{\\svgs/hand.svg}.\n"
+"\\item El número de \\hyperlink{Ability}{Efectos Expertos} \\includesvg[height=10px]{\\svgs/expert.svg} que puedes usar en una Ronda.\n"
+"\\item En qué Niveles tu Héroe Principal debe \\hyperlink{Playerdecks}{Buscar} una nueva \\hyperlink{Ability}{Carta de Habilidad} o ganar una \\hyperlink{Specialty}{Carta de Especialidad}.\n"
+"Los números de nivel escritos en dorado en el seguimiento de niveles (\\includesvg[height=10px]{\\svgs/level1.svg}, \\includesvg[height=10px]{\\svgs/level4.svg} y \\includesvg[height=10px]{\\svgs/level6.svg}) te da una Tarjeta Especializada, mientras que los niveles plateados (2, 3, 5, 7) te da una Carta de Habilidad.\n"
+"\\end{itemize}\n"
+"\\vfill\\null\n"
+"\\columnbreak\n"
+"Lista de los efectos de nivel:\n"
+"\\begin{itemize}\n"
+"\\item \\textbf{Nivel 1} – Tu límite de mano\\index{Hand Limit} es 4.\n"
```
Comment (qwrtln) [inline translations/heroes.tex/es.po:385]: Please translate entries under `index{...}`.

**Review by qwrtln — CHANGES_REQUESTED** [review]
Translate index entries, apart from that looks good 👍🏻

**Comment by luischos** [comment]
Done

**Review by qwrtln — APPROVED** [review]

**Pull request merged**